docs/configuration/forms.md

```diff
@@ -0,0 +1,65 @@
+# Forms
+
+---
+
+When a channel id is set in a form specified in your [config](index.md), it is considered a form channel by the bot. Any messages sent by members without message manage [permission](../discord/perms.md) will be automatically deleted.
```

Comment:
```suggestion
When a channel id is set in a form specified in your [config](index.md), it is considered a Form channel by the bot. Any messages sent by members without the message manage permission will be automatically deleted.
```
Small grammar fix. The permission page is also intended for bot permissions, not general Discord permissions.